src/sentry/api/serializers/models/event.py

```diff
@@ -376,3 +376,36 @@ def serialize(self, obj, attrs, user):
             # Needed to generate minidump links in UI
             "crashFile": attrs["crash_file"],
         }
+
+
+class ExternalEventSerializer(EventSerializer):
+    """
+    Event serializer for the minimum event data needed to send to an external service. This
+    can be used for Integrations and the Integration Platform. If the full event is needed,
```

Comment:
Are there reasons to send different event representations to different services? Where possible shouldn't we attempt to standardise and use this serializer for all integrations and discourage using event.as_dict?

Reply:
Well I think there are cases to use the whole event data, the `error.created` webhook and the data forwarding plugin. So maybe it makes sense to have two external serializers? One with the full payload and one with the simplified payload?

Reply:
Are there only 2 representations of events we will send to external services? Or is it likely we'll want something more custom for different services in future?

Reply:
I'm not sure but integrations usually have some set of required attribute that will change depending on the service, but then there's often an option to add other data (in this case PD has a custom details attribute) and we can just send along the slimmed version of the event in that case. So the entire payload will be custom to the integration, but I don't think we'll need a ton of different customize versions on how an event, which will be part of the payload, looks like if that makes sense.